### PR TITLE
Added week2_classExs CALCULATOR METHOD and DISTINCT APPHABETICAL LIST

### PR DESCRIPTION
**Pull request by sumanghimire79**

### CODE REVIEW

File: week2/week2_classExs/Program.cs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+{
+  var isNumber1Valid = int.TryParse(input, out var parsedInput);
+
+  if (isNumber1Valid)
+  {
+    return Results.Ok(AddNumbers(parsedInput));
+  }
+  return Results.Ok(CountCapitalLetters(input));
+
+  int AddNumbers(int parsedInput)
```
Comment (Vertygo) [inline week2/week2_classExs/Program.cs]: This method is inside the endpoint method. And this kind of method (method inside of the method) is called the [local function](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/local-functions) (yes, it's not called local method 🙈 ). Also, this function can't be called outside of its parent method. We will maybe cover/mention this in our next class.

File: week2/week2_classExs/Program.cs [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+// Make a GET endpoint that takes a string as input and returns 
+// a new list containing only the unique characters, sorted in alphabetical order.
+//  For example, if the input string is The cool breeze whispered through the trees, 
+// the output should be ["b", "c", "d", "e", "h", "i", "l", "o", "p", "r", "s", "t", "u", "w", "z"].
+
+app.MapGet("/3", (string input) =>
+{
+
+  var result = new List<char>();
+  List<char> test_list = new List<char>();
```
Comment (Vertygo) [inline week2/week2_classExs/Program.cs]: In C# it is common to use `camelCase` for the name of variable.

**Review by Vertygo — CHANGES_REQUESTED** [review]
These three are exercises but please finish also homework [homework](https://github.com/HackYourFuture-CPH/dotnet-masterclass/blob/main/week2/homework.md) for this week.

File: week2/week2_classExs/Program.cs (hunk 2 above)
Comment (sumanghimire79) [reply week2/week2_classExs/Program.cs]: i fixed it to camelCase. thank you

**Review by Vertygo — APPROVED** [review]

File: week2/week2_classExs/Program.cs (hunk 1 above)
Comment (sumanghimire79) [reply week2/week2_classExs/Program.cs]: ok !!

**Pull request merged**